modules/solid_mechanics/doc/content/source/materials/CappedWeakPlaneStressUpdate.md

```diff
@@ -1 +1,767 @@
+the Cosserat case (see [CappedWeakPlaneCosseratStressUpdate.md]).
+
+The joint slipping is assumed to be governed by a
+Drucker-Prager type of plasticity with a cohesion $C$, and friction
+angle $\phi$:
+
+\begin{equation}
+f_{0} = q + p\tan\phi - C \ .
+\end{equation}
+
+The parameter $C$ and $\phi$ may be constants, or they may harden or
+soften (more on this later).
+
+Joints may also open, and this type of failure is assumed to be
+governed by a tensile failure yield function:
+
+\begin{equation}
+f_{1} = p - S_{T} \ ,
+\end{equation}
+
+where $S_{T}$ is the tensile strength, which may be constant or harden
+or soften.
+
+Joints may also close, and this type of failure is assumed to be
+governed by a compressive failure yield function:
+
+\begin{equation}
+f_{2} = - p - S_{C} \ ,
+\end{equation}
+
+where $S_{C}$ is the compressive strength (a positive quantity), which
+may be constant or harden or soften.
+
+The yield functions $f_{1}$ and $f_{2}$ place ``caps'' on the shear
+yield function $f_{0}$.
+The combined yield function is simply
+
+\begin{equation}
+f = \max(f_{0}, f_{1}, f_{2}) \ ,
+\label{one.yf.eqn}
+\end{equation}
+
+which defines the admissible domain where all yield functions are
+non-positive, and the inadmissible domain where at least one yield
+function is positive.
+
+
+One of the features of this plasticity is the
+ability to model cyclic behavior. For instance, the compressive
+strength may be initially very high. However, after tensile failure,
+the compressive strength can soften to zero in order to model the fact
+that the material now contains open joints which cannot support any
+compressive load. If the material then fails in compression (eg,
+because it gets squashed) and the joints close then the compressive
+strength can be made high again.
+
+## Flow rules and hardening
+
+This plasticity is non-associative.  Define the dilation angle $\psi$,
+which may be constant, or harden or soften.  The shear flow potential
+is
+
+\begin{equation}
+g_{0} = q + p\tan\psi .
+\end{equation}
+
+The tensile flow potential is
+
+\begin{equation}
+g_{1} = p \ ,
+\end{equation}
+
+and the compressive flow potential is
+
+\begin{equation}
+g_{2} = - p \ .
+\end{equation}
+
+The overall flow potential is
+
+\begin{equation}
+g = \left\{
+\begin{array}{ll}
+g_{0} & \mbox{ if } f = f_{0} \\
+g_{1} & \mbox{ if } f = f_{1} \\
+g_{2} & \mbox{ if } f = f_{2}\ .
+\end{array}
+\label{one.g.eqn}
+\right.
+\end{equation}
+
+Obviously there are problems here where $g$ is not defined properly at
+the corners where $f_{0}=f_{1}$ and $f_{0}=f_{2}$ (or even
+$f_{0}=f_{1}=f_{2}$).  This is resolved by using smoothing (more on
+this later).
+
+This plasticity model contains two internal parameters, denote by
+$i_{0}$ and $i_{1}$.   It is assumed
+that
+
+\begin{eqnarray}
+C & = & C(i_{0}) \ , \\
+\phi & = & \phi(i_{0}) \ , \\
+\psi & = & \psi(i_{0}) \ , \\
+S_{T} & = & S_{T}(i_{1}) \ , \\
+S_{C} & = & S_{C}(i_{1}) \ .
+\end{eqnarray}
```

Comment:
It looks like eqnarray is not supported in `.md`. I looked elsewhere in the repository and it is only used in one case unrelated to this PR: `CombinedScalarDamage.md`, where it also does not render properly.

Reply:
ah that's unfortunate. I ll figure something out